parm/use_cases/model_applications/medium_range/TCStat_SeriesAnalysis_fcstGFS_obsGFS_FeatureRelative_SeriesByInit.conf

```diff
@@ -145,11 +144,14 @@ TC_STAT_STORM_NAME =
 
 # Stratify by init times via a comma-separate list of init times to
 # include or exclude.  Time format defined as YYYYMMDD_HH or YYYYMMDD_HHmmss
-TC_STAT_INIT_BEG = 20141213
-TC_STAT_INIT_END = 20141220
+#TC_STAT_INIT_BEG = 20141213
```

Comment:
Do you want leave these commented out here? Or remove them?

Reply:
I think I mentioned in a Slack chat that there are commented lines throughout that I was planning on cleaning up as I work through the refactor of ExtractTiles and SeriesBy use cases. I also wanted to have a discussion on how these are used. Currently, if any of these TC_STAT_ variables are set in the SeriesBy* use cases, the values will be used for both runs of TC-Stat that are called from ExtractTiles and SeriesBy*. We could also just remove these options from the use case config files and require the user to set any of these values in the job args list specified in EXTRACT_TILES_FILTER_OPTS and SERIES_ANALYSIS_FILTER_OPTS to avoid confusion.